do.sh

```diff
@@ -17,4 +21,17 @@ if ! grep -q "set editing-mode vi" "${HOME}/.inputrc" 2>/dev/null; then
 fi
 
 
+echo "Configuring iptables firewall..."
+iptables -F
+iptables -X
+iptables -P INPUT DROP
```

Comment:
Use a rollback(async) timer before setting the `DROP` policy. and change it to `ACCEP` if the timer is not stopped with the user's confirmation.

Reply:
Done